azure-stack/operator/compare-azure-azure-stack.md

```diff
@@ -26,7 +26,7 @@ This article describes the differences between global Azure, Azure Stack Hub, an
 
 ## Global Azure
 
-Microsoft Azure is an ever-expanding set of cloud services to help your organization meet your business challenges. It's the freedom to build, manage, and deploy apps on a massive, global network using your favorite tools and frameworks.
+Microsoft Azure is an ever-expanding set of cloud services to help your organization meet your business challenges. It's the freedom to build, manage, and deploy apps on a massive, global network using your favourite tools and frameworks.
 
```

Comment:
This is the British spelling of the word, so the original spelling was correct.

Reply:
Hi @sethmanheim,

I was going to let the commit merge and change it back to simplify the contribution.

Thanks,
Pat

Reply:
I applied the edit/reversal and will merge.